sequencer_server/src/record.rs

```diff
@@ -380,7 +380,7 @@ impl MemMapRecordReader {
             if app_ids.is_empty() { 
                 // No whitelist, match all. 
                 // Length tag 
-                writer.write_u16_le(message.len() as u16).await?;
+                writer.write_u64_le(message.len() as LengthTag).await?;
```

Comment:
I'm guessing this is using tokio's `WriteAsyncExt` - is there something equivalent to https://docs.rs/byteorder/latest/byteorder/trait.WriteBytesExt.html#method.write_uint available? Would be nice to only have to change the LengthTag newtype if we change this in the future.

Reply:
From what I can see there is no official Tokio support for it but a crate for this purpose has been published: https://docs.rs/tokio-byteorder/latest/tokio_byteorder/